### DISCUSSION TITLE
How to search skip connection architecture using Auto-Keras

### DISCUSSION THREAD

**Discussion by XiaolongHu1993** [keras-team/autokeras#1315] [General]
How to search skip connection architecture using Auto-Keras like https://github.com/carpedm20/ENAS-pytorch? Can anybody offer some help?

**Comment by haifeng-jin** [comment]
For anything related to the algorithms in the AutoKeras KDD paper. please refer to the 0.4 version and below of AutoKeras. Thanks.
